### PR TITLE
WIP: local dev config for item-bulk-management-service

### PR DESCRIPTION
**Pull request by malaffoon**
Doing a couple at once here:
* item-bulk-management-service
* item-history-service

### CODE REVIEW

File: item-bulk-management-service/README.md [hunk 1]
```diff
@@ -1,6 +1,15 @@
 # AP_ItemBulkManagementService
 Manages bulk updates for items in the TIMS system
 
+## Dependencies
+
+This service requires rabbit to be running.
+TODO - does it really require the permission service?
```
Comment (malaffoon) [inline item-bulk-management-service/README.md]: @bdrainer ?

File: docker-compose.yml [hunk 2]
```diff
@@ -252,27 +252,30 @@ services:
...
     image: fwsbac/tims-item-history-service:latest
-    healthcheck: *default-healthcheck
     ports:
-      - 8086:8080
+      - 8086:8086
+    volumes:
+      - ${HOME}:/home
     depends_on:
       - rabbitmq
-      - item-management-service
```
Comment (malaffoon) [inline docker-compose.yml]: @bdrainer does item-history-service require ims? seems like it uses events so ims is NOT a direct dependency

File: item-bulk-management-service/README.md (hunk 1 above)
Comment (bdrainer) [reply item-bulk-management-service/README.md]: Yes, IBMS directly uses the permission service.  The api gateway and the item authoring tool are the only other two that directly use it.

File: docker-compose.yml (hunk 2 above)
Comment (bdrainer) [reply docker-compose.yml]: No.  Item history does not require IMS.

**Pull request merged**